### DISCUSSION TITLE
Catch--up proposal

### DISCUSSION THREAD

**Discussion by Jasonlovedataanalyst** [CarlosPietrasanta/Tafe230130#8] [General]
Hello All

        How are you today?

         would you think that there would be a good idea for us to catch up sometime on this Sunday, or the other proposed time frame ?

best regards 

Jason Zhang

**Comment by ozdotcom** [comment]
Sunday sounds good for me,

Sharryn

**Comment by CarlosPietrasanta** [comment]
Yep, Sunday is great for me as well. 

See you then.

Carlos.

**Comment by Jasonlovedataanalyst** [comment]
@ozdotcom @CarlosPietrasanta 

Do you have a preferred time frame ,  how about 4 :00 PM to 4:30 PM Sunday  26/3

**Comment by CarlosPietrasanta** [comment]
That will work perfect for me, but I'm happy to change it to whenever works better for everyone!

Let's wait for @ozdotcom confirmation and I'll send the Teams Meeting invitation.

**Comment by ozdotcom** [comment]
Good Morning,

4.00 - 4.30pm works great for me.

@Jasonlovedataanalyst 
@CarlosPietrasanta
